src/main/java/com/BrewMES/demo/API/BrewMESController.java

```diff
@@ -1,11 +1,9 @@
 package com.BrewMES.demo.API;
 
-import com.BrewMES.demo.model.BeerType;
-import com.BrewMES.demo.model.BrewMES;
-import com.BrewMES.demo.model.Command;
-import com.BrewMES.demo.model.iBrewMES;
+import com.BrewMES.demo.model.*;
```

Comment:
Jeg kan ikke helt se hvorfor BrewMESController har brug for alt i model pakken? Skal den ikke kun bruge IBrewMES?

Reply:
Det har du ret i, men jeg tror det hører til et andet issue, fordi vi stadig bruger enums i controlleren. Det blev vi enige om, at fjerne, så BrewMes har ansvaret for det